build.sbt

```diff
@@ -202,15 +201,14 @@ lazy val publishSettings = Seq(
       browseUrl = url("https://github.com/scala-native/scala-native"),
       connection = "scm:git:git@github.com:scala-native/scala-native.git"
     )),
-  pomExtra := (
+  pomExtra :=
```

Comment:
It is good practice to surround XML literals with `()`, due to quirks in how several nodes may or may not be bundled in a `NodeSeq`. I'm surprise IntelliJ flags this. In any case, it would be better to keep the `()`.

Reply:
Well, it was a suggestion. Good to know, will revert.